### PR TITLE
Fix some typos in doc and www

### PR DESCRIPTION
**Pull request by blarz**

### CODE REVIEW

File: www/faq.txt [hunk 1]
```diff
@@ -359,7 +359,7 @@ done
 
 for i in ${!tag_keys[@]} ; do
     key="${tag_keys[$i]}"
-    # they kebinding is as follows: the or executes the commands seperated by
+    # the keybinding is as follows: the or executes the commands separated by
```
Comment (blarz) [inline www/faq.txt]: "is as follows" sounds wrong, too, not sure what is meant here

Comment (t-wissmann) [reply www/faq.txt]: I just want say that I'm about to explain the following `keybind` lines.

File: www/faq.txt [hunk 2]
```diff
@@ -359,7 +359,7 @@ done
...
     # CASE and stops executing them after the first of those succeeds.
     # the first command is: check that we are on monitor 3 and can switch to tag "${mon1_names[$i]}"
     # if only one of these to steps fail, try the second one (and so one).
```
Comment (blarz) [inline www/faq.txt]: two steps?

File: www/faq.txt (hunk 1 above)
Comment (blarz) [reply www/faq.txt]: maybe "works" instead of "is"?

Comment (t-wissmann) [reply www/faq.txt]: Yes!

File: www/faq.txt (hunk 2 above)
Comment (t-wissmann) [reply www/faq.txt]: yes!

File: www/faq.txt (hunk 1 above)
Comment (blarz) [reply www/faq.txt]: Done

File: www/faq.txt (hunk 2 above)
Comment (blarz) [reply www/faq.txt]: Done!

**Pull request merged**